src/main/java/edu/greenblitz/robotName/subsystems/shooter/Mechanism/ShooterMechanism.java

```diff
@@ -42,11 +42,9 @@ private ShooterMechanism(){
 
 
     public void periodic() {
+        pivot.setAngle(-Pivot.getInstance().getAngle().getDegrees());
```

Comment:
why minus

Reply:
because the mechanism 2d has opposite axes